src/locales/fr/battle.json

```
@@ -32,6 +32,7 @@
   "attackMissed": "{{pokemonNameWithAffix}}\névite l’attaque !",
   "attackHitsCount": "Touché {{count}} fois !",
   "rewardGain": "Vous recevez\n{{modifierName}} !",
+  "rewardGainCount": "Vous recevez\n{{count}} {{modifierName}} !",
```

```suggestion
  "rewardGainCount": "Vous recevez\n{{count}} $t(modifierName, {\"count\": {{count}} }) !",
```

For possible plural handling